src/main/java/com/vilgodskaia/movieplatformpetproject/config/aspect/MovieAop.java

```diff
@@ -0,0 +1,74 @@
+package com.vilgodskaia.movieplatformpetproject.config.aspect;
+
+import com.vilgodskaia.movieplatformpetproject.util.DataType;
+import lombok.extern.log4j.Log4j2;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+
+@Component
+@Aspect
+@Log4j2
+public class MovieAop {
+
+    private final String apiPointCutMovieController = "execution(* com.vilgodskaia.movieplatformpetproject.api.movie.MovieRestController.**(..))";
+    private final String apiPointCutStreamingController = "execution(* com.vilgodskaia.movieplatformpetproject.api.streamingplatform.StreamingPlatformRestController.**(..))";
+    private final String apiPointCutMovieOnStreamingController = "execution(* com.vilgodskaia.movieplatformpetproject.api.movieonstreamingplatform.MovieOnStreamingPlatformRestController.**(..))";
```

Comment:
Could we use annotation @RestController instead of adding each rest-controller here?
https://www.baeldung.com/spring-aop-pointcut-tutorial